bndtools.core/src/bndtools/central/Central.java

```diff
@@ -461,6 +465,9 @@ public static void refreshPlugins() throws Exception {
                         l.modelChanged(p);
                 }
 
+                for (RepositoryPlugin rp : changedRepos) {
+                    repositoriesViewRefresher.repositoryRefreshed(rp);
```

Comment:
Wouldn't it be better to just call `repositoriesRefreshed()` and refresh the full view? Then you get just one job instead of N jobs.